src/components/TrendChart.js

```diff
@@ -155,7 +155,7 @@ class TrendChart extends React.Component {
           until={until}
         />
         <div className="mb2 clearfix">
-          <div className="sm-col mb1 sm-m0 fs-10 bold monospace black">
+          <div className="sm-col mb1 sm-m0 fs-14 bold monospace black">
```

Comment:
I think 14px might be a bit big given the rest of the surrounding content's font sizes:

10px
<img width="803" alt="screen shot 2017-08-07 at 11 37 40 pm" src="https://user-images.githubusercontent.com/780941/29055405-5c1a7b3e-7bca-11e7-9d03-f0cd22d6f9b0.png">

12px
<img width="812" alt="screen shot 2017-08-07 at 11 37 57 pm" src="https://user-images.githubusercontent.com/780941/29055406-5c26ad28-7bca-11e7-8346-c4859bcc004e.png">

14px
<img width="796" alt="screen shot 2017-08-07 at 11 38 32 pm" src="https://user-images.githubusercontent.com/780941/29055441-91048c9a-7bca-11e7-8f64-098b825ebcb2.png">

I'd vote upping the font size to 12px for now.

BTW, @brendansudol great work on that alternating X axis label hiding! It looks just as good at 10px as 14px:
<details>
<img src="https://user-images.githubusercontent.com/780941/29055504-dc4ba2ba-7bca-11e7-9256-4b9e10563873.gif">
</details>